src/popupmenu.c

```diff
@@ -416,6 +416,80 @@ pum_under_menu(int row, int col, int only_redrawing)
 	    && col < pum_col + pum_width + pum_scrollbar;
 }
 
+/*
+ * displays text on the popup menu with specific attributes.
+ */
+    static void
+pum_screen_put_with_attr(int row, int col, char_u *text, int textlen, int attr)
+{
+    int		i;
+    int		leader_len;
+    int		char_len;
+    int		cells;
+    int		new_attr;
+    char_u	*rt_leader = NULL;
+    char_u	*match_leader = NULL;
+    char_u	*ptr = text;
+    garray_T	*ga = NULL;
+    char_u	*leader = ins_compl_leader();
+
+    if ((get_cot_flags() & COT_FUZZY) == 0)
+    {
+       screen_puts_len(text, textlen, row, col, attr);
+       return;
```

Comment:
I think we should also skip this, if the HLF_PMSI and HLF_PMNI highlightings are not setup?

Reply:
yup also need compare attr if same as PSI also need run into this if.